### PR TITLE
Finished with news site project (week 2)

### PR DESCRIPTION
**Pull request by Cilpersson**
My website https://vibrant-bardeen-6fca39.netlify.com/

I didn't manage to center my small news cards, I would like to know how to do it :)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,122 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Uplifting News!</title>
+    <link rel="icon" href="images/book.png" />
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Major+Mono+Display&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
```
Comment (ericaglimsholt) [inline code/index.html]: Tip! When selecting fonts in Google Fonts you can choose multiple fonts. Then you will only get one `<link>` to insert here

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,122 @@
...
+      </div>
+
+      <div class="container-upliftingnews">
+        <h1>Uplifting News</h1>
+      </div>
+
+      <nav>
+        <a href="">World</a>
+        <a href="">Local</a>
+        <h1>Uplifting News</h1>
```
Comment (ericaglimsholt) [inline code/index.html]: Try to only use one `<h1>` for each page. The `<h1>` should describe what the page is

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,122 @@
...
+            warbird ezekiel silversurfer magus. She-hulk deadpool patriot.
+          </p>
+        </div>
+      </section>
+      <!-- Other news  -->
+      <div class="small-news-container">
+        <section class="small-news-card">
+          <div>
+            <h3>These are tiny news!</h3>
+            <img src="images/puppy.jpg" alt="Small news image" />
```
Comment (ericaglimsholt) [inline code/index.html]: The reason why we have `alt` as props on images is to describe what the image is on for those that can't see the image. For example "puppy" is a good description when the image is on a puppy

File: code/index.html (hunk 1 above)
Comment (Cilpersson) [reply code/index.html]: I didn't know that, thank you!

**Pull request closed without merging**